packages/mobile/src/notifications.ts

```diff
@@ -54,51 +43,37 @@ const getPlatformConfiguration = () => {
 class PushNotifications {
   lastId: number
   token: Token | null
-  drawerHelpers: DrawerNavigationHelpers | null
-  bottomTabNavigation: Nullable<BottomTabNavigation>
+  navigation: any | null
 
   // onNotification is a function passed in that is to be called when a
   // notification is to be emitted.
   constructor() {
     this.configure()
     this.lastId = 0
     this.token = null
-    this.bottomTabNavigation = null
-  }
-
-  setDrawerHelpers(helpers: DrawerNavigationHelpers) {
-    this.drawerHelpers = helpers
+    this.navigation = null
   }
 
-  setBottomTabNavigation = (bottomTabNavigation: BottomTabNavigation) => {
-    this.bottomTabNavigation = bottomTabNavigation
+  setNavigation = (navigation: any) => {
+    this.navigation = navigation
   }
 
-  onNotification = async (notification: any) => {
+  onNotification = (notification: any) => {
     console.info(`Received notification ${JSON.stringify(notification)}`)
     if (notification.userInteraction || Platform.OS === 'android') {
       track(
         make({
           eventName: EventNames.NOTIFICATIONS_OPEN_PUSH_NOTIFICATION,
           ...(notification.message
             ? {
-                title: notification.message.title,
-                body: notification.message.body
+                title: notification.message.title ?? notification.title,
```

Comment:
oh interesting, is this the new format of notifs?

Reply:
yea the title and body isn't always in message so just covering that case. Can clean this up after more testing probably